httpstan/models.py

```diff
@@ -248,7 +248,7 @@ def _redirect_stderr_to(stream: IO[Any]) -> int:
 
     # write files need for compilation in a temporary directory which will be
     # removed when this function exits.
-    with TemporaryDirectory() as temporary_directory:
+    with TemporaryDirectory('httpstan_') as temporary_directory:
```

Comment:
Maybe match the use above (i.e., with or without `prefix=`)?

(I'm assuming you'll have to change this anyway since black is going to complain.)

Reply:
Yes, there should be `prefix=`.